### PR TITLE
Fix using proper env for platform sidecar statsd

### PR DESCRIPTION
**Pull request by gxueatlassian**
Refer doc: https://hello.atlassian.net/wiki/spaces/MICROS/blog/2022/08/15/1830221772/PSA+We+re+changing+how+platform+sidecar+addresses+are+derived

**What's in this PR?**
Change to use ENV vars

**Why**
Because the fix hostname is gonna be deprecated

**Added feature flags**
N/A

**Affected issues**  
N/A

**How has this been tested?**  
Stage only

Deployed on stage
Screen Shot 2023-02-16 at 12 00 51 pm

Traffic still coming through
Screen Shot 2023-02-16 at 12 01 17 pm

**Whats Next?**

### CODE REVIEW

File: src/config/env.ts [hunk 1]
```diff
@@ -61,7 +61,9 @@ envCheck(
 	"MICROS_AWS_REGION",
 	"GLOBAL_HASH_SECRET",
 	"CRYPTOR_URL",
-	"CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE"
+	"CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE",
+	"MICROS_PLATFORM_STATSD_HOST",
```
Comment (gxueatlassian) [inline src/config/env.ts:65]: Make sure it fails fast

**Comment by bgvozdev** [comment]
Has it been tested in stg?

**Comment by bgvozdev** [comment]
Do we need a similar change for cryptor?

**Review by bgvozdev — APPROVED** [review]

**Comment by gxueatlassian** [comment]
> Has it been tested in stg?

Yup that's the screen shot on description

**Comment by gxueatlassian** [comment]
> Do we need a similar change for cryptor?

May or may not. But cryptor is more explicitly declared, so I guess it is okay.

**Pull request merged**